DecipherNow/gm-fabric-dashboard
# Add Code Climate config and modify CircleCI config

Resolves #969 

## Proposed Changes

  - Add a local codeclimate.yml config
  - Modify the `.circleci/config.yml` to upload Istanbul LCOV reports to Code Climate
  
## How to Test

  - Click the Code Climate Icon for this PR and confirm that the engines are running against project files, but not tests or stories
  - Click on the CircleCI icon to confirm that that test coverage info is getting set to Code Climate. Note: there seems to be no way to see these reports in Code Climate until this PR is merged.
